test/common/dailies.coffee

```diff
@@ -141,19 +140,17 @@ describe 'daily/weekly that repeats everyday (default)', ->
     it 'is due on startDate', ->
       daily_due_today = shared.shouldDo moment(), daily
       daily_due_on_start_date = shared.shouldDo moment().add(7, 'days'), daily
- 
+
       expect(daily_due_today).to.be false
       expect(daily_due_on_start_date).to.be true
 
       weekly_due_today = shared.shouldDo moment(), weekly
       weekly_due_on_start_date = shared.shouldDo moment().add(7, 'days'), weekly
- 
+
       expect(weekly_due_today).to.be false
       expect(weekly_due_on_start_date).to.be true
 
   describe 'when startDate is in the past', ->
-    completeDaily = null
```

Comment:
No need to wait :) Just notify me when it's merged and I will add it to the
next Bailey.

On Sat, Aug 8, 2015 at 11:57 PM, Alys <notifications@github.com> wrote:

> In test/common/dailies.coffee
> <https://github.com/HabitRPG/habitrpg/pull/5756#discussion_r36585089>:
>
> >        expect(weekly_due_today).to.be false
> >        expect(weekly_due_on_start_date).to.be true
> >
> >    describe 'when startDate is in the past', ->
> > -    completeDaily = null
>
> This variable didn't seem to be used anywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5756/files#r36585089>.
>